contracts/src/contracts/Teleporter/AvalancheICTTRouter.sol

```diff
@@ -234,39 +236,33 @@ contract AvalancheICTTRouter is Ownable, ReentrancyGuard, IAvalancheICTTRouter {
         address recipient,
         address feeToken,
         address multiHopFallback,
-        uint256 primaryRelayerFeeBips,
-        uint256 secondaryRelayerFeeBips
+        uint256 primaryRelayerFee,
+        uint256 secondaryRelayerFee
     ) external payable virtual nonReentrant {
         address bridgeSource = tokenToSourceBridge[address(0)];
         DestinationBridge memory destinationBridge =
             tokenDestinationChainToDestinationBridge[destinationChainID][address(0)];
 
-        uint256 primaryFeeAmount = (msg.value * primaryRelayerFeeBips) / 10_000;
-
-        uint256 secondaryFeeAmount = (msg.value * secondaryRelayerFeeBips) / 10_000;
-
         SafeERC20.safeIncreaseAllowance(IERC20(feeToken), bridgeSource, msg.value);
-        WrappedNativeToken(payable(feeToken)).deposit{value: primaryFeeAmount}();
+        WrappedNativeToken(payable(feeToken)).deposit{value: primaryRelayerFee}();
```

Comment:
We shouldn't impose to pay the fee in the native token here. The behavior should be similar to `bridgeERC20`: allow the user to specify what token they want to use to pay the primary fee with `primaryFeeTokenAddress` and let the bridge contract handle the fee.